src/test/scala/files_metadata/FilesTestsUtils.scala

```diff
@@ -208,5 +208,25 @@ object FilesTestsUtils {
         s"${ GetFileMetadataTestsData.API_PREFIX }/$fileUUID"
       )
   }
+  // -- delete files meta --
+  def DeleteFile(
+      ownerUUID: String,
+      fileUUID: String,
+      payload: util.HashMap[String, Any]
+  ): Response = {
+    `given`()
+      .port( 8080 )
+      .contentType( "application/json" )
+      .body( payload )
+      .when()
+      .delete(
+        s"${ DeleteFileTestsData.API_PREFIX }/$ownerUUID/$fileUUID"
+      )
+  }
+
+  def generateDeleteFilePayload(): util.HashMap[String, Any] = {
```

Comment:
Since this is endpoint uses the `DELETE` HTTP verb, it does not receive any payload. Payloads are usually needed in `POST`, `PUT` or `PATCH` endpoints, not in `GET` or `DELETE` ones.